ts/components/dialog/OnionStatusPathDialog.tsx

```diff
@@ -76,11 +77,28 @@ const OnionCountryDisplay = ({ labelText, snodeIp }: { snodeIp?: string; labelTe
   return hoverable;
 };
 
+let reader: Reader<CityResponse> | null;
+const lang = 'en';
```

Comment:
Is the modal always in English? Should it not be based on the set language?

Reply:
Looks like it have some language support. Might be nice to use those if the app has them set and otherwise fallback to English 🤔 https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoLite2-City-Test.json